### ISSUE TITLE
Monotouch MvvmCross

### ISSUE THREAD

**Issue by igonzalez3314** [nberardi/newrelic-mobile-xamarin#2]
Hi Nick,

I'm using MvvmCross v3.

When window.MakeKeyAndVisible is called in AppDelegate FinishedLaunching method, the  ViewDidLoad method of the startup view is called time and time again until app crashed.

If I comment NewRelic.NewRelic.StartWithApplicationToken ("mytoken") sentence in AppDelegate FinishedLaunching the app starts correctly.

Any idea will be appreciated.

Iñigo.

**Comment by markgibaud** [comment]
We are also having this happen, have upgraded to the latest package but no resolution as yet.

**Comment by nberardi** [comment]
I am not familiar with MvvmCross. Could it be doing something that is stepping on NewRelic's toes? The package is very easy to build, I would be curious if an older version of NewRelic works. 

Change the version number in the Makefile

https://github.com/nberardi/newrelic-mobile-xamarin/blob/master/ios/Makefile

First clean by calling `make clean` and then build by typing `make`.

**Comment by igonzalez3314** [comment]
I have tried version 3.311 and the new one 3.343. Same result.

I have notice that application runs correctly in iOS Simulator (6.1 and 7.1 version) but it crashes when it is deployed to iPhone.

**Comment by nberardi** [comment]
I have 3.289 deployed in production with no problems.  Although I am not using MvvmCross either.

**Comment by markgibaud** [comment]
Yes this does seem MvvmCross related - although as Stuart doesn't use New Relic he is not sure what to check either:

https://github.com/MvvmCross/MvvmCross/issues/723

I am going to investigate this further today.

**Comment by markgibaud** [comment]
@nberardi can you confirm that this is working for you with the latest Mono version released?

**Comment by teiles** [comment]
@markgibau, did you resolve this mvvmcross  issue?

**Comment by markgibaud** [comment]
Unfortunately not - we simply removed NewRelic in favour of Raygun, and now also Xamarin Insights. These are better than nothing, but I would still prefer NewRelic to be in there as the dashboard and insights into the data are very good. :-/

New Relic have confirmed that they are discussing Xamarin support but can't confirm that it's on the roadmap.

**Comment by RTodorov** [comment]
I'm facing the same issue here, did you guys have any news about it?

I'm using New Relic on my MvvmCross app to collect crash reports and http monitoring but the interactions monitoring is a key feature.

**Comment by markgibaud** [comment]
Better to see https://github.com/MvvmCross/MvvmCross/issues/723 for any potential resolution to this issue.
